vulcan/_message.py

```diff
@@ -85,3 +88,30 @@ def get(cls, api, date_from=None, date_to=None):
                 message["NadawcaId"]
             )
             yield to_model(cls, message)
+
+    @classmethod
+    def send(cls, api, title, content, teachers_id):
+        teachers = []
+        for teacher in teachers_id:
+            teachers_name = (
+                api.dict.get_teacher_json(int(teacher))["Nazwisko"]
```

Comment:
Repeating 3 times the same code is inefficient. Fetch the teacher object once, and then use the variable.